README.md

```diff
@@ -1,2 +1,319 @@
 # std-toast
 This document scopes out a web platform API for a 'toast' pop-up notification.
+
+## Why?
+
+Modern web applications allow users to complete lots of actions per page,
+which necessitates providing clear feedback for each action.
+Toast notifications are commonly used to unobtrusively provide this feedback.
+
+Many libraries in a variety of frameworks implement a version of toast
+(see [research](./study-group/)),
+but the web has no built-in API to address the use case.
+By providing a toast API as part of the web platform's standard library,
+the web becomes more competitive with other app development platforms,
+and web application developers can spend less of their time and bytes
+on implementing this pattern.
+
+Toasts are also a deceptively-tricky pattern to get right.
+They require special accessibility considerations,
+and scenarios involving multiple toasts need special handling.
+[Not all libraries account for these subtleties](./study-group/).
+By providing a built-in toast control that fully handles these aspects,
+we can level up the typical toast experience for both developers and users of the web.
+
+Finally,
```

Comment:
(I wrote this text.)

Yes, the goal is that other toasts should be based on ours, and e.g. if someone writes a file picker component that wants to use toasts, they can then use std-toast inside of it, instead of worrying about what toast library they need to depend on and whether that makes sense with the application.

Can you think of ways to make this argument stronger?

Reply:
I don't understand your explanation in this comment Domenic. It sounds like you're saying: "by having a standard toast, then other standard APIs can use it." Is that right? If so, I don't really understand what this has to do with what the non-standard toasts do.

Reply:
No, sorry, I was saying that other libraries can use it. Maybe this needs to be spelled out in more detail; it's suffering a bit here by trying to be a short-ish paragraph that can stand alongside the other two as a peer, instead of overshadowing them with a ton of detail.

The scenario I was considering was the following:

- The website author wants to use several complicated third-party controls. One is a file picker. One is a settings panel. One is an email newsletter signup widget.
- All of these different controls want to signal success or completion using toasts.
- All of these different controls are built by different library authors or component vendors.

In the current world, each library author/component vendor would have to pick a toast implementation, or write their own, in order to signal completion. This would lead to the page author likely having three different types of toasts on their page.

In the world with std-toast, each library author/component vendor would use std-toast. Then, when the page author composes the three different widgets onto their page, all the toasts work the same. They can all be styled using `std-toast { ... }` in CSS; they all coordinate to stack on top of each other together instead of overlapping; etc.

This is the "shared vocabulary" argument for standards, just on a UI component level. Like we unified the ecosystem of promises by adding promises to ES2015, or of buttons by adding `<button>` to HTML4, or of disclosure widgets by adding `<details>` to HTML5, the idea is to do the same for toasts.

Make sense?

Reply:
> (I wrote this text.)
> 
> Yes, the goal is that other toasts should be based on ours, and e.g. if someone writes a file picker component that wants to use toasts, they can then use std-toast inside of it, instead of worrying about what toast library they need to depend on and whether that makes sense with the application.
> 
> Can you think of ways to make this argument stronger?

If it's intended that this is a lower layer for all toasts, then I don't think it's a matter of making it stronger, it's more making it clear that that's what it is. E.g. by actually showing that react or whatever can delete some code and use this with no losses.

Reply:
Captured in https://github.com/jackbsteinberg/std-toast/issues/14, for the record